### ISSUE TITLE
eu-south-1 fail scan

### ISSUE THREAD

**Issue by marcobazzani** [cycloidio/terracognita#171]
**General information:**

* Operating System: 
Docker image running on MacOs
* Terracognita version / tag:
cycloid/terracognita                 latest                                                  a3d38f3fc231   8 months ago    407MB

* Did you build Terracognita from sources or did you use the Docker image: 
Docker
**Describe the bug:**
while running 
```
docker run cycloid/terracognita aws --region eu-south-1 `cat .aws/credentials | paste - - - |grep def | awk '{print "--access-key "$4" --secret-key "$7}'` --hcl test.tf -d
```
got this error:

**Log message**
```
ts=2021-03-27T21:38:18.839282862Z caller=import.go:82 func=provider.Import resource=aws_ses_active_receipt_rule_set msg="fetching the list of resources"
Error: could not import from AWS: RequestError: send request failed
caused by: Post "https://email.eu-south-1.amazonaws.com/": dial tcp 15.161.191.156:443: i/o timeout
error while reading from resource "aws_ses_active_receipt_rule_set"
github.com/cycloidio/terracognita/aws.(*aws).Resources
	/app/aws/provider.go:74
github.com/cycloidio/terracognita/provider.Import
	/app/provider/import.go:91
github.com/cycloidio/terracognita/cmd.glob..func2
	/app/cmd/aws.go:86
github.com/spf13/cobra.(*Command).execute
	/go/pkg/mod/github.com/spf13/cobra@v0.0.5/command.go:826
github.com/spf13/cobra.(*Command).ExecuteC
	/go/pkg/mod/github.com/spf13/cobra@v0.0.5/command.go:914
github.com/spf13/cobra.(*Command).Execute
	/go/pkg/mod/github.com/spf13/cobra@v0.0.5/command.go:864
main.main
	/app/main.go:11
runtime.main
	/usr/local/go/src/runtime/proc.go:203
runtime.goexit
	/usr/local/go/src/runtime/asm_amd64.s:1373
github.com/cycloidio/terracognita/provider.Import
	/app/provider/import.go:93
github.com/cycloidio/terracognita/cmd.glob..func2
	/app/cmd/aws.go:86
github.com/spf13/cobra.(*Command).execute
	/go/pkg/mod/github.com/spf13/cobra@v0.0.5/command.go:826
github.com/spf13/cobra.(*Command).ExecuteC
	/go/pkg/mod/github.com/spf13/cobra@v0.0.5/command.go:914
github.com/spf13/cobra.(*Command).Execute
	/go/pkg/mod/github.com/spf13/cobra@v0.0.5/command.go:864
main.main
	/app/main.go:11
runtime.main
	/usr/local/go/src/runtime/proc.go:203
runtime.goexit
	/usr/local/go/src/runtime/asm_amd64.s:1373
could not import from AWS: RequestError: send request failed
caused by: Post "https://email.eu-south-1.amazonaws.com/": dial tcp 15.161.191.156:443: i/o timeout
error while reading from resource "aws_ses_active_receipt_rule_set"
github.com/cycloidio/terracognita/aws.(*aws).Resources
	/app/aws/provider.go:74
github.com/cycloidio/terracognita/provider.Import
	/app/provider/import.go:91
github.com/cycloidio/terracognita/cmd.glob..func2
	/app/cmd/aws.go:86
github.com/spf13/cobra.(*Command).execute
	/go/pkg/mod/github.com/spf13/cobra@v0.0.5/command.go:826
github.com/spf13/cobra.(*Command).ExecuteC
	/go/pkg/mod/github.com/spf13/cobra@v0.0.5/command.go:914
github.com/spf13/cobra.(*Command).Execute
	/go/pkg/mod/github.com/spf13/cobra@v0.0.5/command.go:864
main.main
	/app/main.go:11
runtime.main
	/usr/local/go/src/runtime/proc.go:203
runtime.goexit
	/usr/local/go/src/runtime/asm_amd64.s:1373
github.com/cycloidio/terracognita/provider.Import
	/app/provider/import.go:93
github.com/cycloidio/terracognita/cmd.glob..func2
	/app/cmd/aws.go:86
github.com/spf13/cobra.(*Command).execute
	/go/pkg/mod/github.com/spf13/cobra@v0.0.5/command.go:826
github.com/spf13/cobra.(*Command).ExecuteC
	/go/pkg/mod/github.com/spf13/cobra@v0.0.5/command.go:914
github.com/spf13/cobra.(*Command).Execute
	/go/pkg/mod/github.com/spf13/cobra@v0.0.5/command.go:864
main.main
	/app/main.go:11
runtime.main
	/usr/local/go/src/runtime/proc.go:203
runtime.goexit
	/usr/local/go/src/runtime/asm_amd64.s:1373
Usage:
  terracognita aws [flags]
  terracognita aws [command]

Available Commands:
  resources   List of all the AWS supported Resources

Flags:
      --access-key string   Access Key (required)
  -h, --help                help for aws
      --region string       Region to search in, for now * it's not supported (required)
      --secret-key string   Secret Key (required)
  -t, --tags strings        List of tags to filter with format 'NAME:VALUE'

Global Flags:
  -d, --debug             Activate the debug mode wich includes TF logs via TF_LOG=TRACE|DEBUG|INFO|WARN|ERROR configuration https://www.terraform.io/docs/internals/debugging.html
  -e, --exclude strings   List of resources to not import, this names are the ones on TF (ex: aws_instance). If not set then means that none the resources will be excluded
      --hcl string        HCL output file
  -i, --include strings   List of resources to import, this names are the ones on TF (ex: aws_instance). If not set then means that all the resources will be imported
      --target strings    List of resources to import via ID, those IDs are the ones documented on Terraform that are needed to Import. The format is 'aws_instance.ID'
      --tfstate string    TFState output file
  -v, --verbose           Activate the verbose mode

Use "terracognita aws [command] --help" for more information about a command.
```
Here you can paste the log message or paste the link to console logs. If the log message is too big, you can use a tool like https://pastebin.com/.

**Additional context**

**Comment by xescugc** [comment]
I did a bit of digging.

Found the https://docs.aws.amazon.com/general/latest/gr/ses.html which lists all the endpoints for the SES and there is none for `eu-south-1`.

As it does not exists it returns an error (actually it's a Timeout but nevertheless).

The main issue is that we are "assuming" that all the services are available on all regions, which for what I see is a wrong assumption.

I'll try to find the documentation of which services are available in which regions so we can decide to try to fetch them or not.
